numpy/add_newdocs.py

```diff
@@ -5919,6 +5919,13 @@ def luf(lamdaexpr, *args, **kwargs):
         First array
     B : array_like
         Second array
+    dtype : data-type code, optional
+        The type used to represent the intermediate results. Defaults
```

Comment:
You can check that that is the case by using various combinations of dtype and out.